miden-tx/src/tests/kernel_tests/test_tx.rs

```diff
@@ -820,57 +780,95 @@ fn test_load_foreign_account_basic() {
     );
 
     foreign_account_data_memory_assertions(&foreign_account, &process);
+
+    // GET ITEM TWICE
+    // --------------------------------------------------------------------------------------------
+    // Check the correctness of the memory layout after two consecutive invocations of the
+    // `get_item` account procedures. Invoking two foreign procedures from the same account should
+    // result in reuse of the loaded account.
+
+    let code = format!(
+        "
+        use.std::sys
+
+        use.kernel::prologue
+        use.miden::tx
+
+        begin
+            exec.prologue::prepare_transaction
+
+            {get_item_test_code} 
+            dropw
+            # => []
+
+            # call `get_item` again
+            {get_item_test_code}
+
+            # truncate the stack
+            exec.sys::truncate_stack
+        end
+        ",
+        get_item_test_code = get_item_test_code(&foreign_account_id),
+    );
+
+    let process = tx_context.execute_code(&code).unwrap();
+
+    assert_eq!(
+        try_read_root_mem_value(&process, NATIVE_ACCOUNT_DATA_PTR + ACCOUNT_DATA_LENGTH as u32 * 2),
+        None,
+        "Memory starting from 6144 should stay uninitialized"
+    );
 }
 
-/// This test checks that invoking two foreign procedures from the same account results in reuse of
-/// the loaded account.
```

Comment:
Nit: Can you replace this comment with an updated one? I find those useful to get a quick overview of what the test is doing when one needs to debug it.

Reply:
I moved this comment [here](https://github.com/0xPolygonMiden/miden-base/blob/8dcbb7f8f700ca4e80e6763da99493b5eb001848/miden-tx/src/tests/kernel_tests/test_tx.rs#L786) and slightly updated it to add more context. I decided to merge old tests to one in order to reuse the transaction data, so I separated these tests with comments, which have a small description about the test below.

Reply:
Sure, makes sense as well to do it that way!